index.js

```diff
@@ -6,6 +6,8 @@ const app = express();
 app.listen(3000, () => console.log('Example app listening on port 3000!'));
 app.use(express.static(path.join(__dirname, '/public')));
 
-app.use('/spotify', require('./routes/spotify'));
+app.use('/spotify', require('./routes/routes'));
```

Comment:
This is a bad file name, please revert it back to spotify. The routes are still routes that relate to spotify

Reply:
I was going to make it a general routes file, but that's probably better.  Changing it!